kornia/geometry/liegroup/se2.py

```diff
@@ -55,7 +57,7 @@ def __init__(self, rotation: So2, translation: Tensor) -> None:
     def __repr__(self) -> str:
         return f"rotation: {self.r}\ntranslation: {self.t}"
 
-    def __getitem__(self, idx) -> 'Se2':
+    def __getitem__(self, idx: Union[int, slice]) -> 'Se2':
```

Comment:
Is really slice supported like this ?

Reply:
e.g.

```python
class A:
    b = list(range(100)) 
    def __getitem__(self, x: int):
        return self.b[x]

a = A()
a[1]
a[2:5]
```
if we run mypy:
```bash
$ mypy t.py 
t.py:8: error: Invalid index type "slice" for "A"; expected type "int"  [index]
Found 1 error in 1 file (checked 1 source file)
```

I think it's the same thing for tensors here, no?